### DISCUSSION TITLE
Missing Data / Insurance Company Accountability

### DISCUSSION THREAD

**Discussion by truffledonal** [CMSgov/price-transparency-guide#648] [General]
Going through some bills, I have been checking the quoted rate on the MRFs for United Healthcare. They are missing CPT codes as well as NPI numbers for providers that are in-network. If anyone has a point of contact for a UHC representative of these data, or know where I can collect it, that would be great. 

I have been contacting them several ways, but many are unfamiliar with the legislation and the files themselves. 

How do I get this **legally required public information** from these insurance companies? If they do not comply is there any means of recourse to get CMS involved and hold them accountable? I understand that noncompliant payers could face fines of up to $100 per day for each violation and for each individual affected, with limited exceptions.

**Comment by shaselton-usds** [comment]
Are you sure that you downloaded the correct plan or employer file? There are quite a few different plans/employers that are available on the UHC transparency page.

**Reply by truffledonal** [reply]
For this case, I downloaded all plans for the relevant insurance group I am accessing. It is a self-funded/Third-party-administrator plan. I was able to find the CPT code, however the NPI numbers are missing. The bill indicates the provider my client saw was in network. 

I called in to UHC, and they were able to quote me a rate for the procedure, and that rate is not present in/does not match any of the plans for the relevant insurance group. 

I have other cases where other information is missing such as CPT codes. Many plans only include very few entries (~20 billing codes), so I am unsure where to find the rest of the CPT codes. For example, the following plan only includes about 20 billing codes (different case than the original mentioned above):

2023-01-01_United-HealthCare-Services--Inc-_Third-Party-Administrator_New-York-University-_CSP-943-C865_in-network-rates.json.gz

[You can search in the MRF page for UHC, or this is the download link they provided](https://uhc-tic-mrf.azureedge.net/public-mrf/2023-01-01/2023-01-01_United-HealthCare-Services--Inc-_Third-Party-Administrator_New-York-University-_CSP-943-C865_in-network-rates.json.gz)

I cannot find a point of contact at UHC to discuss their  TiC MRF's. I noted in the UHC TiC FAQ that missing data can be retrieved by reaching out to your "account team," but shouldn't this data be publicly retrievable without an account?

**Reply by JR-CMS** [reply]
If you believe you have observed a violation of the Transparency in Coverage requirements, we encourage you to submit a complaint via our website here: https://www.cms.gov/healthplan-price-transparency/contact-us

**Reply by truffledonal** [reply]
Thank you. I will follow up there if UHC does not clear this up themselves.

**Comment by dgolden** [comment]
**I would like to call out the exemplary support of transparency by BCBS of South Carolina.**

My recommendation is that the next version of the MRF specification should have a required field that provides contact information such as that presented below.

https://www.southcarolinablues.com/web/public/brands/sc/about-us/privacy-and-legal/transparency-in-coverage/ 

Contact us
Price transparency requirements can result in very large file sizes that may, in turn, cause issues while downloading them. If you are experiencing problems, please try a different web browser or check your internet and Wi-Fi speed.

**For questions about an issue with this site or downloading files, please contact us for support by emailing [MRF.Support@bcbssc.com](mailto:MRF.Support@bcbssc.com) or by calling [877-274-1715](tel:877-274-1715).**

Please include this information in your email:

- A detailed description of the question or issue
- Any errors you are receiving, including date and time of error
- Group information for the data you are attempting to download
- Your contact information

**Reply by truffledonal** [reply]
This is great. I also believe this should be mandatory in the next MRF specification. I would certainly rather clear it up with the insurance companies directly rather than file a complaint with CMS - although I will certainly do so if they continue noncompliance. 

Thanks for pointing this out, David!
